src/index.test.js

```diff
@@ -517,6 +518,90 @@ describe('#index', function () {
       });
     });
 
+    it('should use globally defined nameTemplate when it`s not provided in definitions', function() {
```

Comment:
What happens if neither a global nor function specific definition is defined?

Reply:
It does not generate any alarm name, so CF takes case of it

https://github.com/Brightspace/serverless-plugin-aws-alerts/pull/8/files#diff-1fdf421c05c1140f6d71444ea2b27638R129